helpers/utilsHelper/Marshal.go

```diff
@@ -0,0 +1,25 @@
+package utilsHelper
+
+import (
+	"encoding/json"
+
+	"github.com/astaxie/beego/logs"
+	"github.com/udistrital/utils_oas/errorctrl"
+)
+
+func Marshal(in interface{}, out *string) (outputError map[string]interface{}) {
```

Comment:
Documentar...
```suggestion
// Marshal hace tal cosa...
func Marshal(in interface{}, out *string) (outputError map[string]interface{}) {
```

PD: Si es conveniente, mover a [`formatdata`](https://github.com/udistrital/utils_oas/tree/master/formatdata)